topdown/strings.go

```diff
@@ -435,6 +467,7 @@ func init() {
 	RegisterFunctionalBuiltin2(ast.FormatInt.Name, builtinFormatInt)
 	RegisterFunctionalBuiltin2(ast.Concat.Name, builtinConcat)
 	RegisterFunctionalBuiltin2(ast.IndexOf.Name, builtinIndexOf)
+	RegisterFunctionalBuiltin2(ast.IndexOfN.Name, builtinIndexOfN)
```

Comment:
[nit] `RegisterFunctionalBuiltin2` is deprecated, let's use `RegisterBuiltinFunc` instead.